contract_forecast/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "license": "AGPL-3",
     "author": "ACSONE SA/NV," "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/contract",
-    "depends": ["base", "contract", "queue_job"],
+    "depends": ["base", "account", "contract", "queue_job"],
```

Comment:
```suggestion
    "depends": ["contract", "queue_job"],
```
Contract directly depends on account, so it's redundant. So if you want to change the line, I would rather remove the useless base.